Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -1362,7 +1362,37 @@ private string GetCleanValue(string value)
             return value;
         }
 
-        return value.RemoveDiacritics().ToLowerInvariant();
+        return NormalizeNameForSearch(value);
+    }
+
+    private static string NormalizeNameForSearch(string value)
```

Comment:
i dont see a reason for adding this as its own dedicated method.
Just include that in the normalise method